join.html

```diff
@@ -0,0 +1,196 @@
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="shortcut icon" href="./img/favicon.ico" type="image/x-icon" />
+    <link rel="icon" href="./img/favicon.ico" type="image/x-icon" />
+    <title>Real Dev Squad | Join</title>
+    <link rel="stylesheet" href="./css/main.css" />
+    <link rel="stylesheet" href="./css/animate.css" />
+    <link rel="stylesheet" href="./css/navbar.css" />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@100;400;700&display=swap"
+      rel="stylesheet"
+    />
+
+    <!-- Primary Meta Tags -->
+    <meta name="title" content="Real Dev Squad" />
+    <meta
+      name="description"
+      content="Community of developers who collaborate to build Real software."
+    />
+
+    <!-- Open Graph / Facebook Meta Tags -->
+    <meta property="og:type" content="website" />
+    <meta property="og:url" content="https://www.realdevsquad.com/" />
+    <meta property="og:title" content="Real Dev Squad" />
+    <meta
+      property="og:description"
+      content="Community of developers who collaborate to build Real software."
+    />
+    <meta
+      property="og:image"
+      content="https://raw.githubusercontent.com/Real-Dev-Squad/website-www/develop/img/Real-Dev-Squad-preview.png"
+    />
+
+    <!-- Twitter Meta Tag-->
+    <meta property="twitter:card" content="summary_large_image" />
+    <meta property="twitter:site" content="@RealDevSquad" />
+    <meta property="twitter:title" content="Real Dev Squad" />
+    <meta
+      property="twitter:description"
+      content="Community of developers who collaborate to build Real software."
+    />
+    <meta
+      property="twitter:image"
+      content="https://raw.githubusercontent.com/Real-Dev-Squad/website-www/develop/img/Real-Dev-Squad-preview.png"
+    />
+
+    <script src="/js/analytics.js"></script>
+
+    <!-- Global site tag (gtag.js) - Google Analytics -->
+    <script
+      async
+      src="https://www.googletagmanager.com/gtag/js?id=G-YRG12HRZKB"
+    ></script>
+
+    <script>
+      window.dataLayer = window.dataLayer || [];
+      function gtag() {
+        dataLayer.push(arguments);
+      }
+      gtag('js', new Date());
+
+      gtag('config', 'G-YRG12HRZKB');
+    </script>
+  </head>
+
+  <body>
+    <div class="dropdown hide">
+      <ul class="dropdown-list">
+        <li class="dropdown-item">
+          <a class="dropdown-link" href="https://my.realdevsquad.com/">
+            My Profile
+          </a>
+        </li>
+        <hr class="line" />
+        <li class="dropdown-item dropdown-link" id="signout-option">
+          Sign out
+        </li>
+      </ul>
+    </div>
+    <button id="scroll-to-top-btn">
+      <img src="./img/arrow_up.svg" alt="scroll to top button" />
+    </button>
+    <nav>
+      <div class="hamburger">
+        <span class="bar"></span>
+        <span class="bar"></span>
+        <span class="bar"></span>
```

Comment:
Could these be self closing? `<span class="bar" />`

Reply:
I tried to keep the code structure same as the one in Index.html